package.json

```diff
@@ -5,7 +5,8 @@
   "main": "app.js",
   "type": "module",
   "scripts": {
-    "start": "./node_modules/.bin/eslint . && node app.js",
+    "start": "node app.js",
+    "prodstart": "./node_modules/.bin/eslint . && node app.js",
```

Comment:
I think we can remove the `prodstart` script later once the CI is in place checking the linter in the PR. Not a blocker